test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -50,4 +57,17 @@ public AddressBook getTypicalAddressBook() {
         return ab;
     }
 
+    public List<ReadOnlyPerson> getListWithAllPersons() {
+        List<ReadOnlyPerson> list = new ArrayList<ReadOnlyPerson>();
+        for (Person p : getTypicalPersons()) {
+            list.add(new Person(p));
+        }
+        return list;
+    }
+
+    public List<ReadOnlyPerson> getListWithSomePersons() {
+        List<ReadOnlyPerson> list = getListWithAllPersons();
+        list.remove(0); //remove a person in the list
+        return list;
+    }
```

Comment:
This method is very specific to your test class and therefore not a good abstraction. Try something like this instead:
`public List<ReadOnlyPerson> getList(ReadOnlyPerson... personsToInclude)`;

Can be used like this:
`getList(td.amy, td.candy);`